core/src/main/java/io/grpc/inprocess/InProcessChannelBuilder.java

```diff
@@ -48,6 +48,26 @@ public static InProcessChannelBuilder forName(String name) {
     return new InProcessChannelBuilder(name);
   }
 
+  /**
+   * Create a channel builder that will connect to the server with the given name.
+   *
+   * @since 1.7.0
+   * @param target the name of the in process server to connect to.
+   */
+  public static InProcessChannelBuilder forTarget(String target) {
+    return forName(target);
```

Comment:
These should throw. It doesn't provide the target semantics: the name resolver would be ignored.

Reply:
Done.